### ISSUE TITLE
a11y: Landmark role defined incorrectly

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13833]
**a11y issue 46**

**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Landmark role defined incorrectly

The `<main>` element is incorrectly defined twice resulting in a confusing experience for screen reader users. All the page regions should be appropriately defined.

**To Reproduce**

**Result**

**Expected Result**

All the page regions should be appropriately defined.

**Screenshots**

**Additional context**

**Comment by izaac** [comment]
Rancher version: `v2.12-49289cc9c6590b361d64950977dd20b1214908d7-head`
Rancher UI version (dashboard): `master d748be6`

Non-functional change. There are pages like Login the main was mentioned two times. Reproduced on `v2.10.6-alpha6`
main is now mentioned once. 
✅
